### ISSUE TITLE
New home for archive of tutorial materials

### ISSUE THREAD

**Issue by quicklizard99** [NaturalHistoryMuseum/inselect#459]

**Comment by llivermore** [comment]
This Box link should be public: https://nhm.box.com/s/0oo5vaabypxsgt9zzcz7b6o5xg4yjrba

Links need updating on: 
- https://naturalhistorymuseum.github.io/inselect/pages/documentation.html
- https://naturalhistorymuseum.github.io/inselect-SPNHC2016/worksheet.html

**Comment by quicklizard99** [comment]
Inselect site updated.

Annoyingly, the materials for the SPNHC workshop differ slightly from the documentation materials. Feedback from the workshop resulted in the development of the exercises on the Inselect site. Is anybody interested in going through the workshop?

**Comment by llivermore** [comment]
Yes - I got an email about it but I could just point them to the documentation site instead? I have already sent them the files as I presumed they were the same.

**Comment by quicklizard99** [comment]
Yes, would be better all round to refer people to the documentation.
